etc/sudoers.d/spinnaker

```diff
@@ -0,0 +1,2 @@
+# User rules for spinnaker
+spinnaker ALL=(ALL) NOPASSWD:ALL
```

Comment:
One thing is that if we do this for only /usr/bin/packer we should comment or change https://github.com/spinnaker/rosco/blob/master/rosco-web/config/rosco.yml#L7 so that it is clear that there is a convention for where we install packer. If it is set like this it will work like magic on some installations, while others will have a hard time finding out why it is not working.